Classes/ScrollableGraphView.swift

```diff
@@ -332,7 +360,12 @@ import UIKit
         #if TARGET_INTERFACE_BUILDER
             self.offsetWidth = 0
         #else
-        if (direction == .rightToLeft) {
+        var graphSizeExceedsWindow = false
+        if (self.viewportWidth < totalGraphWidth) {
+            graphSizeExceedsWindow = true
+        }
```

Comment:
Btw that part of the code (function setup ()) charges how the graph works when direction is set to left to right and number of items on the graph is less than screen size. Previously it was like a right aligned graph with this code it would be a graph that looks like leftToRight until screen is filled and then it work Would start looking like right aligned graph.

Please don't merge this function of it charges functionality of rightToLeft.